<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Training</class>
 <widget class="QWidget" name="Training">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Тренировка</string>
  </property>
  <widget class="QLabel" name="TimeLeft">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>450</width>
     <height>31</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string>Осталось времени:</string>
   </property>
  </widget>
  <widget class="QLabel" name="TriesLeft">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>450</width>
     <height>31</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string>Осталось попыток:</string>
   </property>
  </widget>
  <widget class="QLabel" name="PointMeter">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>60</y>
     <width>450</width>
     <height>31</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string>Очки:</string>
   </property>
  </widget>
  <widget class="QLabel" name="LabelTime">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>0</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="LabelTries">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>30</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="LabelPoints">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>60</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="Check">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>90</y>
     <width>400</width>
     <height>111</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>что нужно написать</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="Write">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>200</y>
     <width>451</width>
     <height>171</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>90</y>
     <width>1</width>
     <height>311</height>
    </rect>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>449</x>
     <y>90</y>
     <width>1</width>
     <height>311</height>
    </rect>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="PauseLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>370</y>
     <width>450</width>
     <height>30</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string>Esс - пауза/продолжить</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <zorder>PauseLabel</zorder>
  <zorder>TimeLeft</zorder>
  <zorder>TriesLeft</zorder>
  <zorder>PointMeter</zorder>
  <zorder>LabelTime</zorder>
  <zorder>LabelTries</zorder>
  <zorder>LabelPoints</zorder>
  <zorder>Check</zorder>
  <zorder>Write</zorder>
  <zorder>line</zorder>
  <zorder>line_2</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
